Multisig/UI/Settings/OwnerKeyManagement/SeedPhraseViewController/SeedWordCollectionViewCell.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  Copyright © 2019 Gnosis Ltd. All rights reserved.
+//
+
+import UIKit
+
+
+class SeedWordCollectionViewCell: UICollectionViewCell {
+
+    @IBOutlet weak var numberLabel: UILabel!
+    @IBOutlet weak var badgeImageView: UIImageView!
+    @IBOutlet weak var wordLabel: UILabel!
+    @IBOutlet weak var backgroundImageView: UIImageView!
+
+    override func awakeFromNib() {
```

Comment:
Why not set those attributes in the XIB?

Reply:
I copied it from the legacy code and just deleted enough code so that it would work, I'd leave it here like it is if you allow :)

Reply:
Sure :)